src/index.html

```diff
@@ -2,16 +2,175 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible"
+    content="ie=edge"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+    href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset1">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input type="text"
+          name="Surname"
+          id="surname"
+          autocomplete="off"
+          >
```

Comment:
неправильное форматирование
```suggestion
          <input
            type="text"
            name="Surname"
            id="surname"
           autocomplete="off"
          >
```

Reply:
В чеклисті пункти 2,3,4, з візуальними прикладами. Думаю, що було б норм і в інші пункти включити візуальні приклади.